services/src/services/update/update.hooks.js

```diff
@@ -18,6 +19,19 @@ class UpdateHooks {
     return context;
   }
 
+  checkUpdateFormat(hook) {
+    if (!hook) {
+      throw new errors.BadRequest('No hook at all?');
```

Comment:
This check is unnecessary, Feathers hooks are always guaranteed to be called with a hook `context` with a [specified format](https://docs.feathersjs.com/api/hooks.html#hook-context)